src/services/jwt.service.js

```
@@ -0,0 +1,62 @@
+require('dotenv').config();
+
+const jwt = require('jsonwebtoken');
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: process.env.JWT_ACCESS_TTL,
+  });
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch {
+    return null;
+  }
```

The verify/verifyRefresh/verifyReset catch blocks return null on any verification error which is fine, but they swallow the underlying exception. Consider logging the error (in development) to aid debugging when tokens fail verification. Also using `catch (err)` is preferable to a bare `catch` so you can optionally inspect/log the error.